test/verify/machineslib.py

```diff
@@ -1368,10 +1368,20 @@ def installWithErrorTest(dialog):
 
         checkFilteredOsTest(TestMachines.VmDialog(self, storage_size=1, os_name=config.MAGEIA_3_FILTERED_OS))
 
+
         # try to CREATE WITH DIALOG ERROR
         # name
         checkDialogFormValidationTest(TestMachines.VmDialog(self, "", storage_size=1), {"Name": "Name must not be empty"})
 
+        # name already exists
+        createTest(TestMachines.VmDialog(self, name='existing-name', sourceType='url',
+                                         location=config.VALID_URL, storage_size=1,
+                                         delete=False))
+
+        checkDialogFormValidationTest(TestMachines.VmDialog(self, "existing-name", storage_size=1), {"Name": "already exists"})
+
+        self.machine.execute("virsh destroy existing-name")
```

Comment:
This will not undefine the VM though, it will still appear on the list and it will be annoying when debugging future test failures.